backup_and_restore/application_backup_and_restore/installing/installing-oadp-kubevirt.adoc

```diff
@@ -0,0 +1,58 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-oadp-kubevirt"]
+// If docs team approves the name, "Configuring OADP with Kubevirt" great but I suspect it will need to be "Configuring OADP with OpenShift Virtualization" or "Configuring OADP with Virtualization" or perhaps... bear with me.. "OADP and OpenShift-Virtualization"
+= Configuring the {oadp-full} with {VirtProductName}
+include::_attributes/common-attributes.adoc[]
+:context: installing-oadp-kubevirt
+:installing-oadp-kubevirt:
+:credentials: cloud-credentials
+:provider: kubevirt
+
+toc::[]
+
+You can install the {oadp-first} with {VirtProductName} by installing the OADP Operator and configuring a backup location. Then, you can install the Data Protection Application.
+
+// include::snippets/oadp-mtc-operator.adoc[]
+
+Back up and restore virtual machines by using the xref:../../../backup_and_restore/index.adoc#application-backup-restore-operations-overview[{oadp-full}].
+
+[NOTE]
+====
+{oadp-full} with {VirtProductName} supports the following backup and restore storage options:
+
+* Container Storage Interface (CSI) backups
+
+* Container Storage Interface (CSI) backups with DataMover
+
+The following storage options are excluded:
+
+* File System backup and restore.
+
+* Volume Snapshot Backups and Restores.
+
+For more information, see xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-backing-up-applications-restic-doc.adoc#oadp-backing-up-applications-restic-doc[Backing up applications with File System Backup: Kopia or Restic].
+====
+
+
+include::modules/install-and-configure-oadp-kubevirt.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+. xref:../../../backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc#oadp-plugins_oadp-features-plugins[{oadp-short} plugins]
+. xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#backing-up-applications[`Backup` custom resource (CR)]
+. xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#restoring-applications[`Restore` CR]
+. xref:../../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks]
+
+include::modules/oadp-installing-dpa-1-3.adoc[leveloffset=+1]
+
+[IMPORTANT]
```

Comment:
If this is important, why is it the last thing in the assembly? If it is something that a reader needs to know about this process, hoist it up.

Reply:
+1 Max. A support statement is important and lobbing it at the bottom could lead customers down a path they can't actually take in the end.